pcs/src/poly.rs

```diff
@@ -203,7 +203,6 @@ where
 {
     type Output = Self;
 
-    // TODO: (alex) add `Add<'a Self, Output=Self>` and internally use that instead.
```

Comment:
Saves a few clones on the calling site. Arithmetic usage is self-contained in this module AFIACT.